src/openrct2/ride/CableLift.cpp

```diff
@@ -115,10 +115,10 @@ void Vehicle::CableLiftUpdate()
  */
 void Vehicle::CableLiftUpdateMovingToEndOfStation()
 {
-    if (velocity >= -439800)
+    if (velocity >= -6.71082_mph)
```

Comment:
While the value is correctly calculated, this yields in a slightly different number because we're dealing with integers when you go the other way around: -439799.

We could either manually pick a higher value that gives the expected result, leave it and increment the network version, or choose not to use the literal here.

Reply:
I've reverted the non-round constants back.